pubspec.yaml

```diff
@@ -163,34 +167,35 @@ dependencies:
   media_kit_libs_video: ^1.0.1
   video_player: ^2.7.2
   js: ^0.6.7
-  super_clipboard: 0.8.4
+  super_clipboard: 0.8.15
```

Comment:
demo this case for text, image, file

Reply:
https://github.com/linagora/twake-on-matrix/assets/43041967/03e8b4a8-bfb0-4649-b459-dc881d0f3a88